src/components/LoginsPage.tsx

```diff
@@ -0,0 +1,175 @@
+import Head from "next/head";
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+
+import { BoxedLink } from "./display/fewlines/BoxedLink/BoxedLink";
+import { Button, ButtonVariant } from "./display/fewlines/Button/Button";
+import { IdentityContainer } from "./display/fewlines/IdentityContainer/IdentityContainer";
+import { NeutralLink } from "./display/fewlines/NeutralLink/NeutralLink";
+import { Separator } from "./display/fewlines/Separator/Separator";
+import { ShowMoreButton } from "./display/fewlines/ShowMoreButton/ShowMoreButton";
+import { Identity } from "@src/@types/Identity";
+import { SortedIdentities } from "@src/@types/SortedIdentities";
+import { NavigationBreadcrumbs } from "@src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs";
+
+type LoginsProps = {
+  sortedIdentities: SortedIdentities;
+};
+
+const LoginsPage: React.FC<LoginsProps> = ({ sortedIdentities }) => {
+  const [hideSecondaryEmails, setHideSecondaryEmails] = React.useState<boolean>(
+    true,
+  );
+  const [hideSecondaryPhones, setHideSecondaryPhones] = React.useState<boolean>(
+    true,
+  );
+
+  let emailList: Identity[];
+  let phoneList: Identity[];
+
+  hideSecondaryEmails
+    ? (emailList = sortedIdentities.emailIdentities.filter(
+        (identity) => identity.primary,
+      ))
+    : (emailList = sortedIdentities.emailIdentities);
+
+  hideSecondaryPhones
+    ? (phoneList = sortedIdentities.phoneIdentities.filter(
+        (identity) => identity.primary,
+      ))
+    : (phoneList = sortedIdentities.phoneIdentities);
+
+  const { emailIdentities, phoneIdentities } = sortedIdentities;
+
+  return (
+    <Wrapper>
+      <Head>
+        <title>Connect Logins</title>
+      </Head>
+      <IdentitySection>
+        <NavigationBreadcrumbs
+          title="Logins"
+          breadcrumbs="Your emails, phones and social logins"
```

Comment:
This shouldn't be a breadcrumb, but just a small header. Breadcrumbs represent a progression.

Reply:
OK fixed 2762b34

Reply:
Well, header means title in this case. So you can't do it like this :).

```jsx
<h1>Logins</h1>
<h2>Your emails, phones and social logins</h2>
```

If you need a wrapper, you can call it `HeadersWrapper`.

If you look at this:
![image](https://user-images.githubusercontent.com/31956107/94447645-46deb280-01aa-11eb-9c83-a951d7db0fb1.png)

First line is an `h1`, second is an `h3` and third line is an `h2`.

Reply:
Actually if you double check my commit, I did use a `<SmallHeader />` wrapper in which title and subtitle are nested.
As a matter of fact, first line in an `h1` second line is a `p` with a `font-size` of `1.4rem` and third line is an `h3`